playbooks/monitoring/elasticsearch/docs_compare.py

```diff
@@ -47,6 +46,31 @@ def handle_special_case_cluster_stats(internal_doc, metricbeat_doc):
     metricbeat_doc["stack_stats"]["xpack"]["ilm"]["policy_stats"] = new_policy_stats
     metricbeat_doc["stack_stats"]["xpack"]["ilm"]["policy_count"] = len(new_policy_stats)
 
+    # Metricbeat modules will automatically strip out keys that contain a null value
+    # and `license.max_resource_units` is only available on certain license levels.
+    # The `_cluster/stats` api will return a `null` entry for this key if the license level
+    # does not have a `max_resouce_units` which causes Metricbeat to strip it out
+    # If that happens, just assume parity between the two
+    if not "max_resource_units" in metricbeat_doc["license"]:
+      metricbeat_doc["license"]["max_resource_units"] = internal_doc["license"]["max_resource_units"]
```

Comment:
I was thinking something like this:

```suggestion
if 'max_resource_units' in internal_doc['license'] and internal_doc['license']['max_resource_units'] == None:
    internal_doc['license'].pop('max_resource_units')
```